tests/test_scaleway_backend.py

```diff
@@ -0,0 +1,109 @@
+from django.core import mail
+from django.test import SimpleTestCase, override_settings, tag
+
+from anymail.exceptions import AnymailAPIError, AnymailConfigurationError
+
+from .mock_requests_backend import RequestsBackendMockAPITestCase
+
+
+@tag("scaleway")
+@override_settings(
+    EMAIL_BACKEND="anymail.backends.scaleway.EmailBackend",
+    ANYMAIL={"SCALEWAY_API_KEY": "test_api_key"},
+)
+class ScalewayBackendMockAPITestCase(RequestsBackendMockAPITestCase):
+    DEFAULT_RAW_RESPONSE = b"""{
+        "emails": [
+            {
+                "id": "b7c6a8c1-19a4-49d8-9293-41835462ce81",
+                "message_id": "<b7c6a8c1-19a4-49d8-9293-41835462ce81@example.com>",
```

Comment:
Just FYI, in my quick testing, I got two different UUIDs for id and message_id, and the message_id was not formatted as an email *Message-Id*:

```
      "id": "c4a471a9-d195-4dc1-b55e-69cd3dece155",
      "message_id": "1098e9fd-f27f-45f3-9cbe-487ec1dccaec",
```

But the email it sent had this *Message-Id* header, which includes the API response's message_id:

```
Message-Id: <1098e9fd-f27f-45f3-9cbe-487ec1dccaec@phan-van6.scw-tem.cloud>
```